common/scaffolding/interactive-content-en.html

```diff
@@ -0,0 +1,387 @@
+---
+title: Embedded and interactive content - Scaffolding
+---
+---
+{
+	"title": "Embedded and interactive content - Scaffolding",
+	"language": "en",
+	"altLangPage": "interactive-Content-en.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-07-18"
+}
+---
+
+<h2>The <code>picture</code> element</h2>
+<picture></picture>
+<pre><code>&lt;picture&gt;&lt;/picture&gt;</code></pre>
+
+<h2>The <code>source</code> and <code>img</code> element</h2>
```

Comment:
I think it will best if we merge both,

`<picture>`
  - `<source> + <img>`

And then have another `<img>` section alone where at least:
* Valid img src + content in alt
* Empty img src + content in alt
* Invalid img src + content in alt

Reply:
Fixed in commit "Addressed changes".